yarn.lock

```diff
@@ -10782,9 +9362,8 @@ yargs@^8.0.2:
 yargs@~3.10.0:
   version "3.10.0"
   resolved "https://registry.yarnpkg.com/yargs/-/yargs-3.10.0.tgz#f7ee7bd857dd7c1d2d38c0e74efbd681d1431fd1"
-  integrity sha1-9+572FfdfB0tOMDnTvvWgdFDH9E=
   dependencies:
     camelcase "^1.0.2"
     cliui "^2.1.0"
     decamelize "^1.0.0"
-    window-size "0.1.0"
\ No newline at end of file
+    window-size "0.1.0"
```

Comment:
It looks like you're on an older version of Yarn that doesn't have integrity field support. Please try upgrading Yarn and running `yarn` again, you should be able to commit the integrity fields back.